src/components/Player/Pages/Matches/Matches.css

```diff
@@ -0,0 +1,13 @@
+@media only screen and (max-width: 880px) {
+  th[data-mediaq$="_wide"],
+  td[data-mediaq$="_wide"] {
+    display: none;
+  }
```

Comment:
I think this is a good start, but I am not a fan of making predefined breakpoints since we are planning on moving the table outside of the code eventually. Instead, I would rather leverage the redux-responsive to inject the width into the Table and TableHeader components. Then, we can check to see if mediaQuery returns true at that width. I'll post an example in the columns.